File: 2015/src/test/java/Day1Test.java
import nl.sbmf21.aoc15.Aoc;
import nl.sbmf21.aoc15.days.Day1;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day1Test {

    private final Day1 day = new Day1(new Aoc());

    @Test
    public void testPart1Output() {
        assertEquals(138, (int) day.part1());
    }

    @Test
    public void testPart2Output() {
        assertEquals(1771, (int) day.part2());
    }
}

File: 2021/kotlin/src/test/kotlin/Day1Test.kt
import nl.sbmf21.aoc21.Aoc
import nl.sbmf21.aoc21.days.Day1
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class Day1Test {

    private var day = Day1(Aoc())

    @Test
    fun testPart1Output() = assertEquals(1759, day.part1())

    @Test
    fun testPart2Output() = assertEquals(1805, day.part2())
}
